flake.nix

```diff
@@ -10,7 +10,10 @@
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
     flake-utils.url = "github:numtide/flake-utils";
-    nix-npm-buildpackage.url = "github:serokell/nix-npm-buildpackage";
+    nix-filter.url = "github:numtide/nix-filter";
```

Comment:
After we land this we should use `nix-filter` for the deku source to cut down on the rebuilds when changing unrelated files